octopus-energy-rates-card.js

```diff
@@ -231,12 +324,12 @@ class OctopusEnergyRatesCard extends HTMLElement {
     }
 }
 
-customElements.define('octopus-energy-rates-card', OctopusEnergyRatesCard);
+customElements.define('octopus-energy-rates-card', OctopusEnergyRatesCardMv2);
```

Comment:
Any reason for it being called OctopusEnergyRatesCardMv2?

Reply:
I've been running it side by side with the origional so renamed it in my local one, just missed removing it when checking in the change

Reply:
Ah perfect! Working great on mine.